content/basic/typography/index.md

```diff
@@ -137,6 +137,48 @@ function Demo() {
 }
 ```
 
+### 数值组件
+
+基于Text组件，添加了属性: `rule`, `precision`, `truncate`, `parser`, 以提供需要单独处理文本中数值的能力
+
+```jsx live=true
+import React from 'react';
+import { Typography } from '@douyinfe/semi-ui';
+
+function Demo() {
+    const { Numeral } = Typography;
+    return (
+        <div>
```

Comment:
Any example for props.rule='numbers' and 'text' ?  I can't get their behavior from this document right now

Reply:
Roger, I will add it soon

Reply:
![image](https://user-images.githubusercontent.com/88709023/197150684-714cbde4-dda0-4508-a80a-ef5d143df2a5.png)

i push a commit, added more detailed api usage,  you can continue to add here